pkg/virt-operator/resource/generate/components/prometheus.go

```diff
@@ -548,6 +548,23 @@ func NewPrometheusRuleSpec(ns string, workloadUpdatesEnabled bool) *v1.Prometheu
 							operatorHealthImpactLabelKey: "critical",
 						},
 					},
+					{
+						Record: "kubevirt_api_request_deprecated_total",
+						Expr:   intstr.FromString("apiserver_requested_deprecated_apis{group=\"kubevirt.io\"} * on(group,version,resource,subresource) group_right() apiserver_request_total"),
```

Comment:
The expression is incorrect. It will require grouping.

Reply:
@sradco is the `on(group,version,resource,subresource) group_right()` not enough here? I'll try to build an env to play around with this today once https://github.com/kubevirt/monitoring/pull/172 lands.